frontend/src/apis/index.js

```diff
@@ -9,9 +9,13 @@ const myFetch = async (url, config = {}) => {
         credentials: 'same-origin',
         body: JSON.stringify(config.body),
     });
-    if (response.status >= 200 && response.status < 300)
-        return await response.json();
-    else throw new Error('Error in making request');
+    if (response.status >= 200 && response.status < 300) {
```

Comment:
Please always use the strict check operator (=== or !==) to prevent future errors.

Reply:
But isnt this a widely aggreed way to check for response error?

Reply:
Oh sorry, read it wrong. My bad. You are absolutely right!